src/main/java/org/mockito/internal/util/reflection/GenericMetadataSupport.java

```diff
@@ -62,6 +62,42 @@
      */
     protected Map<TypeVariable, Type> contextualActualTypeParameters = new HashMap<TypeVariable, Type>();
 
+    /**
+     * Registers the type variables for the given type and all of its superclasses and superinterfaces.
+     */
+    protected void registerAllTypeVariables(Type classType) {
+        if (!(classType instanceof Class || classType instanceof ParameterizedType)) {
```

Comment:
Codecov complains about not testing this path. I don't think this line is required per https://github.com/JeffreyFalgout/mockito/blob/b4b5b0be26aaf6218cd5b8253faf03e2d116bdb8/src/main/java/org/mockito/internal/util/reflection/GenericMetadataSupport.java#L272-L273 which handles the `instanceof` checks in `inferFrom` which is the only method that uses the `FromParameterizedTypeGenericMetadataSupport`.